README.md

```diff
@@ -127,52 +136,59 @@ For more information, click [here](https://docs.aws.amazon.com/cognito/latest/de
 
 **Retrieving an ID token using aws.cognito.signin.user.admin**
 
-To access the FHIR API, an ID token is required. A Cognito ID token can be obtained using the following command substituting all variables with their values from `INFO_OUTPUT.log` or by using the `serverless info --verbose` command.
-+	For Windows, enter:
+To access the FHIR API, an ID token is required. A Cognito ID token can be obtained using the following command, substituting all variables with values from `INFO_OUTPUT.log` or by using the `serverless info --verbose` command.
++	For Windows, you will need Python 3 installed. Enter:
 ```sh
-scripts/init-auth.py <CLIENT_ID> <REGION>
+python .\scripts\init-auth.py <CLIENT_ID> <REGION>
 ```
-+	For Mac, enter:
++	For Mac/Linux, enter:
 ```sh
 python3 scripts/init-auth.py <CLIENT_ID> <REGION>
 ```
 The return value is the `COGNITO_AUTH_TOKEN` (found in the postman collection) to be used for access to the FHIR APIs.
 
-### Accessing binary resources
+### Accessing Binary resources
 
-Binary resources are FHIR resources that consist of binary/unstructured data of any kind. This could be X-rays, PDF, video or other files. This implementation of the FHIR API has a dependency on the API Gateway and Lambda services, which currently have limitations in request/response sizes of 10 MB and 6 MB respectively. The workaround for this limitation is a hybrid approach of storing a binary resource’s metadata in DynamoDB and using S3's get/putPreSignedUrl APIs. So in your requests to the FHIR API, you will store/get the Binary's metadata from DynamoDB and in the response object it will also contain a pre-signed S3 URL, which should be used to interact directly with the binary file.
+Binary resources are FHIR resources that consist of binary/unstructured data of any kind. This could be medical images, PDFs, video or other files. Because this implementation of the FHIR API uses the API Gateway and Lambda services, which currently have limitations in request/response sizes of 10 MB and 6 MB respectively, the largest request payload that can be sent synchronously through the FHIR interface has been set at 5 MB. Larger payloads use the hybrid approach of storing a binary resource’s *metadata* in DynamoDB and using S3's `get/putPreSignedUrl` APIs. So in your requests to the FHIR API, you will store/get the Binary's metadata in/from DynamoDB and the response payload will contain a pre-signed S3 URL, which should be used to interact directly with the binary file.
```

Comment:
Instead of S3, can we say "Amazon S3"? Recommended for subsequent use: https://w.amazon.com/index.php/AWSDocs/editing/service-names